condition/condition.go

```diff
@@ -4,6 +4,8 @@ import (
 	"reflect"
 	"time"
 
```

Comment:
no newline here. strings should be in the same block of imports as time and reflect